fbfmaproom/fbfmaproom.py

```diff
@@ -845,6 +846,12 @@ def initial_setup(pathname, qstring):
         )
         for k in sorted(c["seasons"].keys())
     ]
+
+    subpages = c.get("subpages", {})
+    season_link_options = [
+        dbc.DropdownMenuItem(subpage, href=f"/fbfmaproom/{subpage_link}")
```

Comment:
We can't hard-code `fbfmaproom` here, because it's configurable, and in fact the production instance is currently mounted at `/fbfmaproom2` (for a time it coexisted with an earlier version that was mounted at `/fbfmaproom`). We need to get the app's mount path from dash (or maybe flask?).